index.src.html

```diff
@@ -749,6 +755,166 @@ <h3 id="try-delivery" algorithm>
   8.  Return "`Failure`".
 </section>
 
+<section>
+  <h2 id="observers">Reporting Observers</h2>
+
+  A <dfn>reporting observer</dfn> observes some types of <a>reports</a> from
+  JavaScript, and is represented in JavaScript by the {{ReportingObserver}}
+  object.
+
+  Each <a>environment settings object</a> has a <dfn>registered reporting
+  observer list</dfn>, which is an <a>ordered set</a> of <a>reporting observers</a>.
+
+  Any <a>reporting observer</a> that is in a <a>registered reporting observer
+  list</a> is considered <dfn>registered</dfn>.
+
+  Each <a>environment settings object</a> has a <dfn>report buffer</dfn>, which
+  is an ordered list of <a>reports</a> that have been generated in
+  that <a>environment settings object</a>. This list is initially empty, and the
+  reports are stored in the same order in which they are generated.
+
+  <h3 id=interface-reporting-observer>Interface {{ReportingObserver}}</h3>
+
+  <pre class="idl">
+[NoInterfaceObject]
+interface ReportBody {
+};
+
+[NoInterfaceObject]
+interface Report {
+  readonly attribute DOMString type;
+  readonly attribute DOMString url;
+  readonly attribute ReportBody? body;
+};
+
+[Constructor(ReportingObserverCallback callback, optional ReportingObserverOptions options)]
+interface ReportingObserver {
+  void observe();
+  void disconnect();
+};
+
+callback ReportingObserverCallback = void (sequence&lt;Report> reports, ReportingObserver observer);
+
+dictionary ReportingObserverOptions {
+  sequence&lt;DOMString> types;
+  boolean futureOnly = false;
```

Comment:
For sake of consistency, s/futureOnly/buffered? 

https://w3c.github.io/performance-timeline/#performanceobserverinit-dictionary

Reply:
Done.